pkg/kube/client.go

```diff
@@ -202,17 +213,26 @@ func (c *Client) Get(namespace string, reader io.Reader) (string, error) {
 	// track of tab widths.
 	buf := new(bytes.Buffer)
 	p, _ := c.Printer(nil, printers.PrintOptions{})
-	for t, ot := range objs {
-		if _, err = buf.WriteString("==> " + t + "\n"); err != nil {
-			return "", err
-		}
-		for _, o := range ot {
-			if err := p.PrintObj(o, buf); err != nil {
-				c.Log("failed to print object type %s, object: %q :\n %v", t, o, err)
+
+	sort.Sort(Objs(objs))
+
+	var lastVk string
+	for _, t := range objs {
+		vk := t.vk
+		object := t.object
+		if vk != lastVk {
```

Comment:
can you explain this change and how it relates to the old code? This looks like a significant behavioral change here to me from a cursory glance.

Reply:
The old codes has two level to store all objs: objs stores different type of objes, and each element in objs(var ot) stores the same type of obj(var o).
The new codes have one flat level to store all objs, and after sorting, all these objs are sorted by their types(v1/Service, v1/Secret, etc), so I need `lastVk` to record the last type, and when type changes, printing a new line and the type header like `==> v1/Service`.